Makefile

```diff
@@ -515,6 +520,13 @@ $(done_dir)/safety_install_$(pymn)_$(PACKAGE_LEVEL).done: $(done_dir)/develop_$(
 	echo "done" >$@
 	@echo "Makefile: Done running Safety for install packages"
 
+$(done_dir)/bandit_$(pymn)_$(PACKAGE_LEVEL).done: $(done_dir)/develop_$(pymn)_$(PACKAGE_LEVEL).done Makefile $(check_py_files)
+	@echo "Makefile: Running Bandit"
+	-$(call RM_FUNC,$@)
+	bandit -s B101 -r -l $(package_name) $(mock_package_name)
```

Comment:
minor: add a comment what "-s B101" is doing?

Reply:
I cannot explain every option of every command with a comment.
Maybe we should start using a config file for bandit (in which it is much easier to have explanatory comments).

Reply:
If you think that there will be many changes to these options, that might be a good solution.
But if this is rather static (and unlikely to be changed frequently), I would keep it as is.

Reply:
There is always the possibility of changes, and commenting it would indeed be good. I have added a bandit config file.